debsources/tests/test_web_cp.py

```diff
@@ -202,6 +202,60 @@ def test_api_search_filename_all(self):
             "/copyright/api/file/gnubg/all/doc/gnubg/gnubg.html/").data)
         self.assertEqual(len(rv['result']), 3)
 
+    def test_batch_api(self):
+        data = {"checksums": ["2e6d31a5983a91251bfae5aefa1c0a19d8ba3cf601d0e"
+                              "8a706b4cfa9661a6b8a",
+                              "4f721b8e5b0add185d6af7a93e577638d25eaa5c34129"
+                              "7d95b4a27b7635b4d3f",
+                              "sha_does_not_exist"]
+                }
+        rv = json.loads(self.app.post("/copyright/api/sha256/",
+                                      data=data).data)
+        self.assertEqual("2e6d31a5983a91251bfae5aefa1c0a19d8ba3cf601d0e"
+                         "8a706b4cfa9661a6b8a", rv['result'][0]['checksum'])
+        self.assertEqual(len(rv['result'][0]['copyright']), 12)
+        self.assertEqual(rv['result'][2]['return_code'], 404)
```

Comment:
I'm not sure I like the 404 here. I understand it represents the absence of results, but it's different than an http 404 error (because the http return code of this query would still be 200).
Same goes for 200. Any way to avoid the confusion?